Bottom: 4a3b1885aed958692024b7d0d2c75e43826c29fb
Top:    9d97a05d5690b67e0864484bcf3ffeead96929a1
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2015-01-11 13:39:26 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
